### PR TITLE
Fix local setup issue: Use config.AWS_S3_IS_LOCAL initialized via env variables

### PR DESCRIPTION
**Pull request by tuxdna**

### CODE REVIEW

**Comment by msrb** [comment]
@tuxdna you need to set your membership in f8a GH organization to "public", otherwise CI won't test your PRs 😉

**Comment by msrb** [comment]
[test]

**Comment by tuxdna** [comment]
@msrb: Thanks for pointing that out. Update membership to public now ;-)

File: src/data_importer.py [hunk 1]
```diff
@@ -163,6 +162,7 @@ def import_epv_http(data_source, list_epv, select_doc=None):
                 ver_list_keys = list(set(ver_list_keys).intersection(set(select_ver_doc)))
                 pkg_list_keys = list(set(pkg_list_keys).intersection(set(select_pkg_doc)))
 
+            dict_keys = {}
             dict_keys[pkg_key_prefix] = {
```
Comment (jpopelka) [inline src/data_importer.py]: It's OK, but write it as dictionary literal please:

```
 dict_keys = {pkg_key_prefix: {
                'ver_list_keys': ver_list_keys,
                'ver_key_prefix': ver_key_prefix,
                'pkg_list_keys': pkg_list_keys,
                'pkg_key_prefix': pkg_key_prefix,
                'package': epv.get('name'),
                'version': epv.get('version', ''),
                'ecosystem': epv.get('ecosystem')
                }
            }
```

BTW: One could wonder why the `list_keys` is a list of dictionaries, where every dictionary contains only one key (and the value is again a dictionary) when the key is actually **not** used in the `_import_keys_from_s3_http()`.
So the `list_keys` could be simple list of dictionaries, but well, that's not part of this PR, so just FYI

Comment (tuxdna) [reply src/data_importer.py]: @jpopelka Agreed with your point. However this is only a refactor of existing code i.e. I moved `dict_keys = {}` closer to where it is actually used.
